utils/build.ps1

```diff
@@ -828,6 +828,21 @@ function Get-PinnedToolchainVersion() {
   throw "PinnedVersion must be set"
 }
 
+function Get-ClangDriverName([Platform] $Platform, [string] $Lang) {
```

Comment:
Should we enumerate the languages? Perhaps that is overly complicating the state.

Reply:
Not sure what you mean, a for-loop with if-else instead of a 2-level switch?

Reply:
Hm, possible